tests/spark/test_spark_model_export.py

```diff
@@ -51,24 +90,25 @@ def test_model_export(tmpdir):
     assert preds1 == preds2
     preds3 = score_model_in_sagemaker_docker_container(model_path=str(model_path), data=pandas_df)
     assert preds1 == preds3
+    assert os.path.exists(sparkm.DFS_TMP)
+    print(os.listdir(sparkm.DFS_TMP))
+    assert not os.listdir(sparkm.DFS_TMP)
 
 
-@pytest.mark.large
+# @pytest.mark.large
```

Comment:
Probably add this back. You can run large tests locally by using `pytest --large -k test_spark_model_export`

Reply:
oops, good catch. yeah I did not want to run the other large test which runs sagemaker in docker container. I'll rever it :)